tasks/telemetry.yml

```diff
@@ -0,0 +1,34 @@
+---
+# TODO: @adam.schneider - Make this generic
+- name: Build APM library installation payloads for telemetry
+  set_fact:
+    telemetry_payload:
+      "{{ telemetry_payload | default([]) |
+          combine({'specified_' + item.split(':')[0] + '_lib_version':
+            item.split(':')[1] | default('default')}) }}"
+  loop: "{{ datadog_apm_instrumentation_libraries }}"
```

Comment:
is the item name js or nodejs? because there is one case where the  name of the cur_lib doesn't match:

```        datadog-apm-library-js)
          library_telemetry_info="${library_telemetry_info},\"specified_nodejs_lib_version\": \"${cur_version}\""
```

Reply:
Fixed. Using a map to get specific tag based on the library specified.